examples/leafwing.rs

```diff
@@ -34,51 +41,156 @@ struct Player {
+                    ..default()
+                },
                 ..default()
             },
-            ..default()
-        },
-    ));
+        ))
+        .with_children(|parent| {
+            // pointy "nose" for player
+            parent.spawn(SpriteBundle {
+                transform: Transform {
+                    translation: Vec3::new(15., 0., 0.),
+                    rotation: Quat::from_rotation_z(PI / 4.),
+                    ..default()
+                },
+                sprite: Sprite {
+                    color: Color::ORANGE,
+                    custom_size: Some(Vec2::splat(50. / f32::sqrt(2.))),
+                    ..default()
+                },
+                ..default()
+            });
+        });
 
-    // spawn a touch stick
-    commands.spawn((
-        // map this stick as a left gamepad stick (through bevy_input)
-        // leafwing will register this as a normal gamepad
-        TouchStickGamepadMapping::LEFT_STICK,
-        TouchStickUiBundle {
-            stick: TouchStick::<MyStick>::default(),
-            // configure the interactable area through bevy_ui
-            style: Style {
-                width: Val::Px(150.),
-                height: Val::Px(150.),
-                position_type: PositionType::Absolute,
-                left: Val::Percent(50.),
-                bottom: Val::Percent(15.),
+    // spawn a move stick
+    commands
+        .spawn((
+            // map this stick as a left gamepad stick (through bevy_input)
+            // leafwing will register this as a normal gamepad
+            TouchStickGamepadMapping::LEFT_STICK,
+            TouchStickUiBundle {
+                stick: TouchStick {
+                    id: Stick::Left,
+                    stick_type: TouchStickType::Fixed,
+                    ..default()
+                },
+                // configure the interactable area through bevy_ui
+                style: Style {
+                    width: Val::Px(150.),
+                    height: Val::Px(150.),
+                    position_type: PositionType::Absolute,
+                    left: Val::Percent(15.),
+                    bottom: Val::Percent(5.),
+                    ..default()
+                },
                 ..default()
             },
-            ..default()
-        },
-    ));
+        ))
+        .with_children(|parent| {
+            parent.spawn((
+                TouchStickUiKnob,
+                ImageBundle {
+                    image: asset_server.load("knob.png").into(),
+                    style: Style {
+                        width: Val::Px(75.),
+                        height: Val::Px(75.),
+                        ..default()
+                    },
+                    ..default()
+                },
+            ));
+            parent.spawn((
+                TouchStickUiOutline,
+                ImageBundle {
+                    image: asset_server.load("outline.png").into(),
```

Comment:
this wont load the correct file on linux/*bsd

Reply:
are you on macos or windows?
these dont work properly if i run them. :0

Reply:
Yes i thought I'd already renamed them. But apparently not

Reply:
Files renamed.